features/fetch.yml

```diff
@@ -2,6 +2,12 @@ name: Fetch (initial support)
 description: The `fetch()` method makes asynchronous HTTP requests.
 spec: https://fetch.spec.whatwg.org/
 caniuse: fetch
+status:
+  compute_from:
+    - api.fetch
```

Comment:
This single entry would be enough I think, but listing a few more doesn't change anything. We'll need some kind of guidance for what people should prefer, however.

Reply:
I was on the fence about this myself, but here's a possible heuristic:

I looked at the first few code samples on the MDN overview page for the feature. On some of the samples, `Headers` preceded `fetch()`, so I figured: where's the first place you're likely to hit a non-available feature? That should go in `compute_from`.

Though I guess now that I've said this, `Response` could probably be dropped. Feels… impure to leave just one interface out though.

Reply:
What's wrong with just `compute_from: api.fetch`?